app/javascript/mastodon/components/avatar.tsx

```diff
@@ -59,7 +61,7 @@ export const Avatar: React.FC<Props> = ({
       style={style}
     >
       {src && !error && (
-        <img src={src} alt={account?.get('avatar_description') ?? ''} onLoad={handleLoad} onError={handleError} />
```

Comment:
Will this image still be inside a link (which should be a button)? In this case the alt attribute of the image should describe the action done by this button and not the image itself. I would propose to add this description on the image inside the modal window.